AGENTS.md

```
@@ -0,0 +1,154 @@
+  - [ ] Criterion 1
+  - [ ] Criterion 2
+  
+references:
+  - docs/features/FEATURE.md
+  - src/types/feature.ts
+```
+
+### Git Strategy
+
+**Branch Naming:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)